src/coreclr/src/gc/gcee.cpp

```diff
@@ -16,9 +16,14 @@ uint64_t g_TotalTimeInGC = 0;
 uint64_t g_TotalTimeSinceLastGCEnd = 0;
 
 uint32_t g_percentTimeInGCSinceLastGC = 0;
+uint64_t g_TimeBetweenLastTwoGCs = 0;
 
 size_t g_GenerationSizes[total_generation_count];
 size_t g_GenerationPromotedSizes[total_generation_count];
+uint64_t g_GenerationTimeBetweenGCs[total_generation_count];
```

Comment:
It looks like total_generation_count has a slot for LOH and POH.  Does the gen2 number include both of those, and we're just creating arrays here that are slightly larger than is actually needed / used?  Or does the gen2 counter not end up reflecting LOH and POH?

Reply:
Oops, this should be `max_generation`. Thanks for catching this!